src/fmu/tools/extract_grid_zone_tops.py

```diff
@@ -7,18 +7,24 @@
 
 try:
     import _roxar  # type: ignore
+    from _roxar import Project  # type:ignore
 except ModuleNotFoundError:
+    try:
+        import _rmsapi as _roxar  # type: ignore
+        from _rmsapi import Project  # type:ignore
+    except ModuleNotFoundError:
 
-    class MockRoxar:
-        @property
-        def Project(self):
-            pass
+        class MockRoxar:
+            @property
+            def Project(self):
+                pass
 
-    _roxar = MockRoxar()
+        _roxar = MockRoxar()
+        Project = _roxar.Project
```

Comment:
It seems to me that all of this is related to type hinting the `project` argument in the function. Could we just remove it and put `Any` as type instead as we do in the `qcreset.py` 🙂 ? for simplicity?

Reply:
I was thinking of that, but left it as it was your code initially; :-) I can change it to Any